meta-refkit-industrial/recipes-industrial/packagegroups/packagegroup-industrial.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "PROJECTNAME Industrial package groups"
```

Comment:
Should you also swap PROJECTNAME for IoT Reference OS Kit (as you did in the refkit-image-industrial.bb file)?